apps/snitch_core/lib/core/data/model/variant.ex

```diff
@@ -3,10 +3,22 @@ defmodule Snitch.Data.Model.Variant do
   Variant API
   """
   use Snitch.Data.Model
+
   alias Snitch.Data.Schema.Variant
 
   def get_category(%Variant{} = v) do
```

Comment:
@oyeb IMO this method should be renamed as `get_shipping_category`. It is not obvious from the name of the function which category it returns.